spec.bs

```diff
@@ -579,46 +580,77 @@ An auction config is a [=struct=] with the following items:
 <dl dfn-for="auction config">
 : <dfn>seller</dfn>
 :: An [=url/origin=].
+  The origin of the seller running the ad auction.
 : <dfn>decision logic url</dfn>
 :: A [=URL=].
+  The URL to fetch the seller's JavaScript from.
 : <dfn>trusted scoring signals url</dfn>
 :: Null or a [=URL=].
+  Provide a mechanism for making real-time data available for use at scoring time.
 : <dfn>interest group buyers</dfn>
 :: Null or a [=list=] of [=url/origin=].
+  Owners of interest groups allowed to participate in the auction.
 : <dfn>auction signals</dfn>
 :: Null or a [=string=].
+  Opaque JSON data passed to both sellers' and buyers' script runners.
 : <dfn>seller signals</dfn>
 :: Null or a [=string=].
+  Opaque JSON data passed to the seller's script runner.
 : <dfn>seller timeout</dfn>
-:: A duration. Defaulting to 50.
+:: A duration. Defaulting to 50 milliseconds.
+  Restricts the runtime of the seller's scoreAd() script.
 : <dfn>per buyer signals</dfn>
 :: Null or an [=ordered map=] whose [=map/keys=] are [=url/origins=] and whose [=map/values=] are
   [=strings=].
+  [=map/Keys=] are buyers and must be valid HTTPS origins. [=map/Values=] are opaque JSON data
+  passed to corresponding buyer's script runner.
 : <dfn>per buyer timeouts</dfn>
 :: Null or an [=ordered map=] whose [=map/keys=] are [=url/origins=] and whose [=map/values=] are
-  durations.
+  durations. 
+  [=map/Keys=] are buyers and must be valid HTTPS origins. [=map/Values=] restrict the runtime of
+  corresponding buyer's generateBid() script.
 : <dfn>all buyers timeout</dfn>
-:: A duration. Defaulting to 50.
+:: A duration. Defaulting to 50 milliseconds.
+  Restricts generateBid() script's runtime of all buyers with unspecified timeouts.
 : <dfn>per buyer group limits</dfn>
 :: Null or an [=ordered map=] whose [=map/keys=] are [=url/origins=] and whose [=map/values=] are
   non-negative 16-bit integers.
+  [=map/Keys=] are buyers and must be valid HTTPS origins. [=map/Values=] restrict the number of
+  bidding interest groups for a particular buyer that can participate in an auction.
+: <dfn>all buyers group limit</dfn>
+:: A non-negative 16-bit integer.
+  Limit on the number of bidding interest groups for any buyer with unspecified limit.
 : <dfn>per buyer priority signals</dfn>
 :: Null or an [=ordered map=] whose [=map/keys=] are [=url/origins=] and whose [=map/values=] are
   [=ordered maps=], whose [=map/keys=] are [=strings=] and whose [=map/values=] are {{double}}.
+  Per-buyer sparse vector that, along with a similar per-interest group sparse vector
+  [=interest group/priority vector=], has its dot product taken to calculate interest group

```

Comment:
```suggestion
  Per-buyer sparse vector whose dot product with [=interest group/priority vector=] is used to calculate interest group
```
It's a little odd to call a thing with string keys a "vector", but I think it's clear enough what you mean.